QueueManagerTest/SearchManagerTests.cs

```diff
@@ -80,5 +80,10 @@ public void TestSearchMultipleItemsWhenASpaceIsPresentWihTwoWords()
             List<string> listToSearch = new List<string>() { "The word", "alpha add", "beta sdf" };
             Assert.AreEqual(new List<string>() { "The word" }, _searchManager.SearchFor(listToSearch, "the wo"));
         }
+        [TearDown]
```

Comment:
So in the future, you don't forget to put in the method if you need it.  It's for consistency's sake.